support/openharmony/.gitignore

```diff
@@ -0,0 +1,4 @@
+# ignore generated files
+.hvigor
```

Comment:
In principle there shouldn't be any generated files here, because ./mach copies this directory into the `target` directory before invoking hvigor. Did you perhaps open this folder in DevEco?
I guess it doesn't harm adding the ignore though.